frontend/packages/gitops-plugin/src/components/list/GitOpsList.tsx

```diff
@@ -21,6 +21,13 @@ const GitOpsList: React.FC<GitOpsListProps> = ({ appGroups, emptyStateMsg }) =>
     return fuzzyCaseInsensitive(textFilter, name);
   });
 
+  let hasSyncStatus: boolean = false;
+  if (appGroups.length > 0) {
+    const appGroup = appGroups[0];
+    if (appGroup.sync_status) {
+      hasSyncStatus = true;
+    }
+  }
```

Comment:
Any reason you don't want to loop all? You could simplify this if you look for the first one:

```suggestion
  let hasSyncStatus: boolean = appGroups?.some(({ sync_status }) => sync_status) || false;
```

Also `appGroups` can be null, best use optional chaining: `?.`

Reply:
Done